libs/server/Resp/ArrayCommands.cs

```diff
@@ -269,15 +269,21 @@ private bool NetworkDEL<TGarnetApi>(ref TGarnetApi storageApi)
         /// <summary>
         /// SELECT
         /// </summary>
-        /// <param name="ptr"></param>
         /// <returns></returns>
-        private bool NetworkSELECT(byte* ptr)
+        private bool NetworkSELECT()
         {
-            // Read index
-            if (!RespReadUtils.ReadIntWithLengthHeader(out var result, ref ptr, recvBufferPtr + bytesRead))
-                return false;
+            if (parseState.count != 1)
```

Comment:
How about checking these counts based on arity in a common way across all commands?

Reply:
Yup, that's on the to-do list